pkg/controller/secret/secret_controller.go

```diff
@@ -579,6 +630,39 @@ func secretInList(name string, list *corev1.SecretList) bool {
 	return false
 }
 
+// cmInList takes the name of ConfigMap, and a list of ConfigMaps, and returns a Bool
+// indicating if the name is present in the list
+func cmInList(name string, list *corev1.ConfigMapList) bool {
```

Comment:
Can you add some unit tests around these functions in secret_controller_test.go?

Reply:
[Unit tests added](https://github.com/openshift/configure-alertmanager-operator/pull/186/files#diff-3f3b1c34e4d392411c2522d11db21d7d89b9603de69a11f63686c664558bb375R387-R429)

Reply:
Thanks!